functions/ts/kubeval/README.md

```diff
@@ -1,10 +1,24 @@
 # kubeval
 
-### Overview
+## Overview
 
 The `kubeval` function wraps the [`kubeval`] binary to validate KRM resources
 against their [json schemas].
 
+This function is often used in the following scenarios:
+
+- Validating KRM resources as part of the local development workflow.
```

Comment:
I wonder if we have to use `KRM` term ? I think calling `resources` should be enough, right ?